### PR TITLE
Refactored Home page

### PR DESCRIPTION
**Pull request by himadri2110**
- Made separate css files for navbar and footer
- Changes to navbar styling

### CODE REVIEW

**Review by manojkumar8400 — APPROVED** [review]
looks good to me.

File: css/footer.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/* Footer */
+
+footer {
```
Comment (msdeshmukh009) [inline css/footer.css:3]: Maybe avoid styling elements directly. I know in this case it makes sense to style the footer element directly but still, it's not good practice.

**Review by msdeshmukh009 — APPROVED** [review]
Looks good!!

File: css/footer.css (hunk 1 above)
Comment (himadri2110) [reply css/footer.css:3]: Yes, I won't be reusing the footer element anywhere else, so it won't be an issue here. 
Thanks for suggesting, I'll keep that in mind!

**Pull request merged**